README.md

```diff
@@ -34,20 +34,41 @@
     - WebSocket: `wss://subgraph-ws.perp.fi/subgraphs/name/perpetual-protocol/perpetual-v2-optimism-goerli`
     - Healthcheck: `https://subgraph-hc.perp.fi/graphql`
 
+### Satsuma
+
+- Optimism
+    - Dashboard: https://app.satsuma.xyz/subgraphs/119
```

Comment:
Should we also put endpoints here just for convenience?

Reply:
The endpoint changes every time we deploy a new version (version name is in url), so I guess we just use the dashboard as the single source of truth.